doc/sources/guide/packaging-android-windows.rst

```diff
@@ -0,0 +1,212 @@
+
+How to build android package on Windows
+=======================================
+
+These are the instructions on how to build Kivy apps for android on "Windows' Ubuntu Bash Shell"
+
+
+If you need instructions on how to activate "Ubuntu Bash Shell" on Windows 10 follow  the instructions on this site:
+
+`How to install and use the linux bash shell on Windows 10 <http://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/>`_
+
+
+The original instructions on how to install p4a are posted on:
+
+
+https://python-for-android.readthedocs.io/en/latest/quickstart/
+
+
+I have made some adjustments to make it work with "Windows' Ubuntu Bash Shell"
+
+
+The docs for p4a, can be found on that site too.
+
+
+These instructions are tested on a fresh install "Ubuntu Bash Shell for Windows 10"::
+
+    $ uname -a
+    Linux LAPTOP 3.4.0+ #1 PREEMPT Thu Aug 1 17:06:05 CST 2013 x86_64 x86_64 x86_64 GNU/Linux
+    $ lsb_release -a
+    No LSB modules are available.
+    Distributor ID: Ubuntu
+    Description:    Ubuntu 14.04.5 LTS
+    Release:        14.04
+    Codename:       trusty
+
+
+.. _Setup windows:
+
+Setup windows
+-------------
+
+First make sure your windows pc is setup right.
+
+The first thing to do, is to change the swap memory size.
+Otherwise there may be out of memory issues while building.
+
+Since the swap size is shared from Windows' virtual memory, we need to change that.
+
+To change the virtual memory on Windows, follow these steps:
+
+    #. Right click on "My Computer"
+    #. Choose "Properties"
+    #. Choose "Advanced system settings"
+    #. Choose "Advanced"
+    #. Choose "Settings" under performance
+    #. Choose "Advanced"
+    #. Choose "Change" under Virtual Memory
+    #. Choose "Custom size" and set it to 8000 MB.
+    #. Press "Ok", then Alpply all changes.
+
+
+The Windows pc might need a reboot after this.
+
+Now that Windows is setup correctly, we can start bash.
+
+
+.. _Install the dependecies:
+
+Install the dependecies.
+------------------------
+
+From now on, everything is done in "Ubuntu Bash Shell".
+
+So open Bash. And execute the following commands.
+
+Everything will be run as root, so start by entering this command::
+
+    $ sudo su
+    
+
+When in the root shell, run these commands::
+
+    # apt-get update
+    # apt-get upgrade -y
+    # apt-get install -y python-pip
+    # pip install python-for-android
+    # dpkg --add-architecture i386
+    # apt-get update
+    # apt-get install -y build-essential ccache git zlib1g-dev python2.7 \
+                                  python2.7-dev libncurses5:i386 libstdc++6:i386 cython \
+                                  zlib1g:i386 unzip ant ccache python-virtualenv \
+                                  autoconf automake pkg-config libtool
+    # add-apt-repository ppa:webupd8team/java -y
+    # apt-get update
+
+
+Install java, and accept license::
+
+    # apt-get install -y oracle-java8-installer oracle-java8-set-default
```

Comment:
I had some troubles with java errors in the process. Thats why I switched.
But that did not fix the errors. It was because of windows virtual memory space was too small.
That said, openjdk might work. I did not test it tho.